packages/wallet/src/cip30.ts

```diff
@@ -172,7 +181,7 @@ export const createWalletApi = (
       if (!rewardAccount) {
         throw new ApiError(APIErrorCode.InternalError, 'could not get reward address');
       } else {
-        return [rewardAccount.toString()];
+        return [cardanoAddressToCbor(rewardAccount)];
```

Comment:
Is this a breaking change? Considering the return value was already specified as `Promise<Cbor[]>`?

Reply:
Yes, it should be pointed out in the commit message. It was a bug, wouldn't have happened if Cbor was an OpaqueString :sweat_smile:

Reply:
Should we add an OpaqueString now then?

Reply:
I will mark it resolved after final squash with commit message updated

Reply:
Should we make a task to convert the alias to an `OpaqueString`?

Reply:
> I will mark it resolved after final squash with commit message updated
Updated the commit message, thus marking this as resolved.